src/Mod/PartDesign/PartDesignTests/TestHelix.py

```diff
@@ -0,0 +1,143 @@
+#***************************************************************************
+#*   Copyright (c) 2023 <bgbsww@gmail.com>                                 *
+#*                                                                         *
+#*   This program is free software; you can redistribute it and/or modify  *
+#*   it under the terms of the GNU Lesser General Public License (LGPL)    *
+#*   as published by the Free Software Foundation; either version 2 of     *
+#*   the License, or (at your option) any later version.                   *
+#*   for detail see the LICENCE text file.                                 *
+#*                                                                         *
+#*   This program is distributed in the hope that it will be useful,       *
+#*   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+#*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+#*   GNU Library General Public License for more details.                  *
+#*                                                                         *
+#*   You should have received a copy of the GNU Library General Public     *
+#*   License along with this program; if not, write to the Free Software   *
+#*   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+#*   USA                                                                   *
+#*                                                                         *
+#***************************************************************************
```

Comment:
It's not important for you to fix it here, but for future reference this is the old version of the license header. You can find the new one here: https://freecad.github.io/DevelopersHandbook/codeformatting/#c-copyright-header

Reply:
Thanks, I'll get that right on all future submissions.  I won't change it here unless something else drives further changes to this PR.